README.md

```diff
@@ -182,7 +183,7 @@ for sponsoring the project.
 - [`fold`: Coerce success and failure values into the same success value](#fold)
 - [`ap`: Combine the success values of multiple Futures using a function](#ap)
 - [`and`: Logical *and* for Futures](#and)
-- [`or`: Logical *or* for Futures](#or)
+- [`alt`: Logical *or* for Futures](#alt)
```

Comment:
Is deleting this a breaking change? If so, why not leave both?

Reply:
I did leave both, I left "or" as an alias to alt (I added mention about this in the documentation of `alt`), it's just that I want "alt" to be leading.